### PR TITLE
✅ Add Keycloak setup docs

### PR DESCRIPTION
**Pull request by deifyed**
Tested and works.

Feel free to test it yourself.

### CODE REVIEW

File: docs/keycloak.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Configure the client with your <gatekeeper url>/callback as "Valid Redirect URIs" and your frontend URL as "Web Origins"
+- The DISCOVERY_URL is the endpoint "OpenID Endpoint Configuration" found on the "Realm Settings" page. (Right click and copy link)
+
+### Create and configure the Client
+
+#### Create the client
+1. Log in to Keycloak and select your realm.
+2. Select "Clients" in the menu on the left and click "Create" in the top right corner of the screen.
+3. Enter a client ID. Note this down, this is the CLIENT_ID. Ensure the Client Protocol is "OpenID Connect".
+4. Click "Save"?.
```
Comment (yngvark) [inline docs/keycloak.md:17]: `?.` ?

File: docs/keycloak.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## TL;DR:
+- Create a client and set its "Access Type" to "Confidential". The name of the client is the CLIENT_ID and the CLIENT_SECRET is in the credentials tab of the client.
+- Configure the client with your <gatekeeper url>/callback as "Valid Redirect URIs" and your frontend URL as "Web Origins"
+- The DISCOVERY_URL is the endpoint "OpenID Endpoint Configuration" found on the "Realm Settings" page. (Right click and copy link)
+
+### Create and configure the Client
+
+#### Create the client
+1. Log in to Keycloak and select your realm.
```
Comment (yngvark) [inline docs/keycloak.md:14]: Github markdown lists supports lists being written as
```
1. Hey
1. Ho
1. Britney
```
"Proof": 

1. Hey
1. Ho
1. Britney

Then you don't need to re-write numbers if you change orders or whatnot.

**Review by yngvark — APPROVED** [review]
Didn't test, but LGTM.

File: docs/keycloak.md (hunk 1 above)
Comment (deifyed) [reply docs/keycloak.md:17]: Thanks! I put a ? there to remember to verify the text xD

File: docs/keycloak.md (hunk 2 above)
Comment (deifyed) [reply docs/keycloak.md:14]: That totally f's up my OCD

**Pull request merged**